src/img/img6.svg

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 26.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!-- Generator: Adobe Illustrator 23.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
How come this file shows up in the diff?

Reply:
I just resized it to better fit the container it was in. 
This made is easier to work with when putting it in the top section.